README.md

```diff
@@ -364,6 +364,38 @@ You can now submit a new commit/change that will update the CI system to run a n
 
 It might happen that the CI does not get reloaded so you can simply leave a comment on your PR with "test this please" and it will trigger a new CI build.
 
+# Backporting changes
+
+If a change should be backported to a `stable-*` Git branch:
+
+* Mark your PR with the GitHub label "Backport" so we don't lose track of it.
+* Determine the latest available stable branch:
+  `git branch -r | grep origin/stable`
+* Switch to the stable branch: `git checkout stable-2.2`
+* Cherry-pick your change: `git cherry-pick -x (your-sha1)`
+* Create a new pull request against the `stable-2.2` branch.
+* Ensure that your PR's title has the prefix "backport:", so it's clear
+  to reviewers what this is about.
+
+All changes to the stable branches should land in master first, so we avoid
+regressions.
+
+Once this is done, one of the project maintainers will tag the tip of the
+stable branch with your change. For example:
+
+```
+git checkout stable-2.2
+git pull --ff-only
+git tag v2.2.5
+git push origin v2.2.5
+```
+
+You can query backport-related items in GitHub:
+* [all PRs marked as backport candidates](https://github.com/ceph/ceph-ansible/labels/backport)
+  to see what work remains
+* [all closed PRs](https://github.com/ceph/ceph-ansible/pulls?q=is%3Apr+is%3Aclosed)
```

Comment:
It is closed when it is merged, so I don't think we can leave it open.
I've updated the queries in my PR here, and added more explanations. Are they ok now?